cookbooks/bach_spark/recipes/default.rb

```diff
@@ -45,7 +55,7 @@
   helpers(Spark::Configuration)
 end
 
-template "#{spark_bin_dir}/conf/spark-defaults.conf" do
+template "/etc/spark2/conf/spark-defaults.conf" do
```

Comment:
Looks like `/etc/spark2/conf` could be a new attribute (something like `node['spark']['conf']['dir']`)?